frontend/components/base/base-render-markdown/RenderMarkdown.base.component.vue

```diff
@@ -27,6 +27,9 @@ DOMPurify.addHook("beforeSanitizeAttributes", (node) => {
       node.setAttribute("viewBox", `0 0 ${width} ${height}`);
     }
   }
+  if (node.tagName === "A") {
```

Comment:
You've probably already checked it out but it's not possible to check if the `node` is of type anchor or something similar instead of comparing to a string?

Reply:
I mean something like this:

```suggestion
  if (node instanceof HTMLAnchorElement) {
```

Reply:
Great idea Jose, I'll try it!